Flixable_com_Scraper_.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>On 2nd thought (previous comment deleted), the best way is probably to let <code>requests</code> handle building the URL for us instead of us doing the work with <code>.format</code> or <code>+</code>. It does this way better and way cleaner. With this, we can add/exclude parameters by changing the dictionary and the changes will flow through without messing around with templated strings to format or concat.</p><pre class="ql-syntax" spellcheck="false">url = "https://flixable.com"
params = {'min-rating': 0,
          'min-year': 0,
          'max-year': 3000,
          'page': 1}

response = requests.get(url, params=params)

# Show the URL that <code>requests</code> built
# Output: https://flixable.com/?min-rating=0&amp;min-year=0&amp;max-year=3000&amp;page=1
print(response.url)
response.text
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AdamSpannbauer/flixable_ml_dsi/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>